### PR TITLE
Sun Html Components

### PR DESCRIPTION
**Pull request by LucioPelinson**

### CODE REVIEW

File: src/Core/Core.csproj [hunk 1]
```diff
@@ -85,6 +85,11 @@
       <Compile Update="UI\Components\IComponentFactoryOfT.cs">
         <DependentUpon>IComponentFactory.cs</DependentUpon>
       </Compile>
+      <Compile Remove="UI\Components\Html\IHtmlComponentFactory.cs" />
+      <Compile Remove="UI\Components\Html\ValidationSummaryFactory2.cs" />
```
Comment (gumbarros) [inline src/Core/Core.csproj]: .csproj noise, can be removed

File: src/Core/UI/Components/Html/HtmlComponentFactory.cs
Comment (gumbarros) [inline src/Core/UI/Components/Html/HtmlComponentFactory.cs:1]: Great

File: src/Core/UI/Components/Html/TabNav/NavFactory.cs [hunk 2]
```diff
@@ -0,0 +1,19 @@
+using JJMasterData.Core.UI.Components;
+using JJMasterData.Core.Web.Http.Abstractions;
+
+namespace JJMasterData.Core.Web.Components;
+
+public class NavFactory : IComponentFactory<JJTabNav>
+{
+    private readonly IHttpContext _httpContext;
+
+    public NavFactory(IHttpContext httpContext)
```
Comment (gumbarros) [inline src/Core/UI/Components/Html/TabNav/NavFactory.cs:10]: Here we can use `IFormValues`, but fine to me `IHttpContext`

**Review by gumbarros — APPROVED** [review]

File: src/Core/Core.csproj (hunk 1 above)
Comment (LucioPelinson) [reply src/Core/Core.csproj]: Done.

**Pull request merged**